src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -329,9 +331,18 @@ private void visitScript(NodeTraversal t, Node script) {
 
       String moduleName = inputToModuleName(t.getInput());
 
-      // Rename vars to not conflict in global scope.
-      NodeTraversal.traverseEs6(compiler, script, new SuffixVarsCallback(
-          moduleName));
+      boolean hasExports = !(moduleExportRefs.isEmpty() && exportRefs.isEmpty());
+
+      // Rename vars to not conflict in global scope - but only if the script exports something.
```

Comment:
So if two files both have
```js
var name = require('name');
```
but neither of them exports anything, would it be a name conflict?